### ISSUE TITLE
Update and publish Python files

### ISSUE THREAD

**Issue by Dreamsorcerer** [mailgun/gubernator#175]
When trying to use the Python files (https://github.com/mailgun/gubernator/tree/master/python/gubernator), I see errors about needing to use protobuf <= 3.20. We also get a lot of type errors when checking with mypy.

Would it be possible to get get these files updated?

Would it also be possible to publish them as a package on PyPI?

**Comment by Baliedge** [comment]
Yes, the Python client is very much outdated.  We could use some help bringing up to date for Python 3 in a PR.
